version1/03 - Code lists.md

```diff
@@ -226,3 +226,27 @@ No default values are present in this list and must be configured in the system
 |:-----|:----------------------|:--------|
 | IMER | Intern merknad        |         |
 | MER  | Merknad               |  &nbsp; |
+
+No default values are present in this list and must be configured in the system before use.
+
+### presedensStatus (M056)
+
+| Code | Name                  | Comment |
+|:-----|:----------------------|:--------|
+| G    | Gjeldende             |         |
+| F    | Foreldet              |  &nbsp; |
+
+No default values are present in this list and must be configured in the system before use.
+
+### presedensHjemmel (M311)
+
+| Code | Name | Comment |
+|:-----|:-----|:--------|
+|      |      |  &nbsp; |
+No default values are present in this list and must be configured in the system before use.
```

Comment:
An empty line is needed before this one for better formatting.

Reply:
Addressed with https://github.com/documaster/noark5-web-services/pull/50/commits/3e7fa81e3c1996d2a7488b335d719518ca15bed6